onnxruntime/core/providers/webnn/builders/model_builder.cc

```diff
@@ -420,6 +425,36 @@ void ModelBuilder::AddOperand(const std::string& name, const emscripten::val& op
   wnn_operands_.insert(std::make_pair(name, operand));
 }
 
+// Get the zero scalar constant.
+// Workaround for builer.constant(value, type) method since it has not been implemented now.
```

Comment:
There's even an issue to remove it from the spec: [https://github.com/webmachinelearning/webnn/issues/475](https://github.com/webmachinelearning/webnn/issues/475)

Reply:
Yeah, we should remove this comment once the spec drop the `builder.constant(value, type)` method.